src/components/repository/RepositoryInfo.vue

```diff
@@ -2,20 +2,26 @@
   <div class="bh-grid">
     <div class="bh-grid__item bh-grid__item--grow-0 repository-info__big-badge-wrapper">
       <div class="repository-info__big-badge">
-        <bh-icon
+        <bh-icon-button
           size="medium"
           value="botinho"
-          class="repository-info__big-badge__icon" />
+          class="repository-info__big-badge__icon"
+          @click="redirect()" />
       </div>
     </div>
     <div class="bh-grid__item">
       <div class="repository-info__title">
         <span class="repository-info__title__bagde">
-          <bh-icon
+          <bh-icon-button
             value="botinho"
-            class="repository-info__title__bagde__icon" />
+            class="repository-info__title__bagde__icon"
+            @click="redirect()" />
         </span>
-        <span>{{ repository.name }}</span>
+        <div
+          class="repository-info__cursor-pointer"
```

Comment:
BH has a class called clickable.. Remove this class definition and set .clickable here.